Flight Software/QT RP2040/Velocity data/code.py

```diff
@@ -51,12 +49,13 @@
         accel_sign = -1
     else:
         accel_sign = 1
-    vel_1 = accel_0 + (accel_sign * accel_mag - g - accel_0) * (time_stamp - time_0) * 0.5
-    time_0 += time_stamp
-    vel_0 += vel_1
-    accel_0 += (accel_mag * accel_sign)
+    accel_1 = accel_mag * accel_sign
+    vel_1 = vel_0 + (accel_1 + accel_0 - 2 * g) * (time_1 - time_0) * 0.5
+    time_0 = time_1
+    vel_0 = vel_1
+    accel_0 = accel_1
```

Comment:
Why are you setting variables to other variables, why not just reference the original?

Reply:
It'll still work. I'll change it once the code is proven.